techniques/silverlight/SL14.html

```diff
@@ -103,9 +103,9 @@ <h3>KeyNumericUpDown Control That Handles Arrow Key Equivalence
     						helper functions (Increment() and Decrement()). </p>
             <p>Handling the + and - keys as alternate or additional keyboard equivalents
     						for the actions is also possible (if that is desired, handler would
-    						check for <code class="prop">Key.Add</code> or <code class="prop">Key.Subtract</code> values). </p>
+    						check for <code>Key.Add</code> or <code>Key.Subtract</code> values). </p>
             <p>The following is the XAML-defined control template for this control.</p>
-            <pre xml:space="preserve">   &lt;Style TargetType="local:KeysNumericUpDown"&gt;
+            <pre>   &lt;Style TargetType="local:KeysNumericUpDown"&gt;
```

Comment:
```suggestion
            <pre><code>   &lt;Style TargetType="local:KeysNumericUpDown"&gt;
```
and add matching `</code>`